randovania/games/prime3/logic_database/Bryyo - Reptilicus.txt

```diff
@@ -98,7 +98,7 @@ Extra - asset_id: 15814834557214288722
   > G.F.S. Theseus Upper Ledge
       Any of the following:
           Morph Ball Bomb and Morph Ball and After G.F.S. Theseus Satellite
-          # https://youtu.be/eHJKUTiMgO8&t=75
+          # Jump on the lip near the ledge and morph at the apex of the jump https://youtu.be/inJ7_pTqlg0
```

Comment:
Why Morph in the comment? It's not listed and you definitely don't need it here.